### ISSUE TITLE
Triage of older issues, some suggestions

### ISSUE THREAD

**Issue by dtig** [WebAssembly/simd#58]
As the SIMD proposal has changed champions in the last few months, and some open issues are pending on performance numbers, there's been a backlog that hasn't been addressed. To handle that I'd like to propose a couple of ideas:

1. Create a new label "Pending performance experiments", and apply them to proposed instructions in the issues/PRs so they can be addressed with use cases, and differentiated from other issues that need feedback.
2. Schedule a one-time meeting to triage some of the older issues in this repository as well as the meetings repository, and discuss follow-up issues if any.

I'll follow up on both of these if there are no strong objections on this issue. CC @PeterJensen @aretmr

**Comment by aretmr** [comment]
Regarding the "Pending perf experiments" categorization, we need to track this for instruction groups and not individual instructions. (based on the CG consensus we had on this topic https://github.com/WebAssembly/simd/issues/37 )

How about creating Issues tracking perf evaluation of each "instruction groups" and linking all related discussions(PR's/Issues) on instructions within the group. The benefit is to have all  considerations related to one particular group organized at a single place.  
Identifying the instruction groups will be a good topic for the triage-meeting. 

Thoughts?

**Comment by dtig** [comment]
Agree that it makes sense to evaluate instruction groups together, but after a quick glance at the open issues I'm not sure if they really fit into a specific set of instruction groups, so there might be use in a  generic pending label to separate them from issues that need attention. Ok to punt this discussion to the meeting as well.

To those interested, please fill out this [**_form_**](https://goo.gl/forms/GPKXZbnETmLUSwFv2) to be sent a link to the triage meeting.

**Comment by aretmr** [comment]
@ditg can you make the above form accessible externally?

**Comment by dtig** [comment]
Sorry about that, done!

**Comment by arunetm** [comment]
created an issue https://github.com/WebAssembly/simd/issues/63 for the sync for better visibility.

**Comment by dtig** [comment]
Closing as a duplicate of #63.
